ynr/apps/bulk_adding/tests/test_bulk_add.py

```diff
@@ -88,7 +88,8 @@ def test_submitting_form(self):
         response = response.follow()
         form = response.forms[1]
         form['form-0-select_person'].select('_new')
-        response = form.submit()
+        with self.assertNumQueries(70):
```

Comment:
+1 for `assertNumQueries()` - making assertions about application performance is really useful.

Simultaneously, this seems like a bit of an arbitrary number. As a reader of this file in isolation, I have no idea why this is 70 (or 66 in the other test). I think these tests could benefit from some sort of explanation of these numbers.

It also seems like this is a test that is going to break quite often. I'm not sure I have a great suggestion for improvement here. One thing I have done in https://github.com/DemocracyClub/uk-geo-utils/blob/master/uk_geo_utils/tests/test_addressbase_geocoder.py is express the performance requirements in terms of a range. It works there because the range is meaningful though. I just feel like this is a number we're often going to find ourselves arbitrarily incrementing/decrementing as we modify code in tangentally related places.. Maybe performance assertions would be best reserved for the more granular unit test in `ynr/apps/bulk_adding/tests/test_helpers.py` where there are less moving parts rather than a high level integration test like this. Do you have any thoughts?

Also, am I reading this correctly that it takes 70 round trips to the database to insert one candidate, and _that's an improvement_ ? That poor database! :( Presumably (looking at the other tests) that's 61 queries of setup/bootstrapping that must occur regardless of whether we are inserting 1 candidate or 50 and then 9 queries per candidate inserted? That's the kind of info that would at least be useful in a comment here.